app/controllers/api/v1/reservations_controller.rb

```diff
@@ -1,14 +1,21 @@
 class Api::V1::ReservationsController < ApplicationController
   def index
+    list_reservations = Reservation.includes(car: [:images]).all
+      .where(user_id: current_user_id).as_json(include: { car: {
+                                                 only: %i[
+                                                   name description price
+                                                 ], include: :images
+                                               } })
+
     render json: {
       data: {
-        reservations: Reservation.all
-      }
-    }, status: :ok
+        reservations: list_reservations
+      }, status: :ok
+    }
```

Comment:
- I think `status: :ok` is meant to apply 200 status code to response. However it is rendered as json property along with reservation list.